ethpm/exceptions.py

```diff
@@ -23,6 +23,14 @@ class ValidationError(PyEthPMError):
     pass
 
 
+class CannotHandleURI(PyEthPMError):
+    """
+    Raised when the given URI cannot be served by any of the available backends.
+    """
+
+    pass
+
+
 class UriNotSupportedError(ValidationError):
     """
     Raised when URI scheme is invalid or not supported by the current backend.
```

Comment:
I think this docstring needs to be updated.  The *not supported* seems to now be covered by the `CannotHandleURI` exception.  Thoughts?

Reply:
Yeah true, there is a subtle difference between the two, but probably not worth the confusion between having two separate but similar exception classes, and the subtle difference is better explained through each instance's error message.